game/scripts/npc/items/item_bloodstone.txt

```diff
@@ -8,14 +8,10 @@
   {
 
     "ID"                                                  "3400"       // unique ID
-    "Model"                                               "models/props_gameplay/recipe.vmdl"
-    "AbilityTextureName"                                  "item_recipe"
-    "BaseClass"                                           "item_datadriven"
```

Comment:
Why did you remove these lines? Add them back and everything will start working. You need a baseclass.